client/src/components/molecules/ClientRequestTable.tsx

```diff
@@ -0,0 +1,229 @@
+import { gql, useMutation } from "@apollo/client";
```

Comment:
Per Figma, part of this should be in an atom

Reply:
sorry, I'm not sure what this means